### PR TITLE
OCPBUGS-20572: Obsolete protocols for webhook

### PR DESCRIPTION
**Pull request by iurygregory**
Set minimum TLS version for webhook to 1.2

### CODE REVIEW

**Comment by zaneb** [comment]
There's no `NewServer()` function in controller-runtime 0.12, you have to do it like https://github.com/openshift/baremetal-operator/pull/297/files

**Comment by iurygregory** [comment]
Ok, seems like the failures are real, still trying to understand what is happening

**Comment by iurygregory** [comment]
@zaneb when you have some time can you help me out in this one?  Thanks!

File: main.go [hunk 1]
```diff
@@ -81,13 +82,17 @@ func main() {
...
 		MetricsBindAddress: metricsAddr,
+		WebhookServer: &webhook.Server{
+			Port:          9443,
+			TLSMinVersion: "1.2",
+		},
 		NewCache: cache.MultiNamespacedCacheBuilder(
 			[]string{controllers.ComponentNamespace, provisioning.OpenshiftConfigNamespace}),
 		LeaderElection: enableLeaderElection,
-		Port:           9443,
 		CertDir:        "/etc/cluster-baremetal-operator/tls",
```
Comment (zaneb) [inline main.go:96]: This has to be moved to the WebhookServer as well.

**Review by zaneb — COMMENTED** [review]
Exciting news, controller-runtime was updated to 0.15 in #383.

File: main.go (hunk 1 above)
Comment (iurygregory) [reply main.go:96]: Ack, I will update things

**Pull request merged**